# fix a error in the main
# By Ward Poelmans <wpoely86@gmail.com>
diff -ur c37b2.orig/source/charmm/charmm_main.src c37b2/source/charmm/charmm_main.src
--- c37b2.orig/source/charmm/charmm_main.src	2013-02-11 06:27:24.000000000 +0100
+++ c37b2/source/charmm/charmm_main.src	2013-12-03 16:30:45.539579132 +0100
@@ -1267,7 +1267,7 @@
 
   case('PIPF','PFBA') cmds
 ##IF PIPF
-     if (wrd == 'PIPF') cmds
+     if (wrd == 'PIPF') then
         call pfprep(comlyn,comlen)
      else if (wrd == 'PFBA') then
         call nosepf(comlyn,comlen)
